### ISSUE TITLE
[6.x] Death Knight zone

### ISSUE THREAD

**Issue by Atis6** [TrinityCore/TrinityCore#14928]
Hi!
In quest Death comes from on high (id : 12641) doesn't works. When you clicks to the gameobject, who shows you the 'blue world', It's crashing the server (I cant paste a log). 
I searched what's can crash the server, and I found. The spell, what casts on you the gameobject (Spell id : 51888) that crashes the server.
Thanks.
Rev: https://github.com/TrinityCore/TrinityCore/commit/881ebccdc4a0995732aea2052f85015aebe6f9a2

**Comment by aaplmike** [comment]
I didn't get a crash, but here's what I saw with the quest...

The quest http://www.wowhead.com/quest=12641/death-comes-from-on-high allows you to drive around the Eye of Archerus, however won't allow you to "analyze" any of the four target buildings. Also, there are no NPCs in the zone, where there are usually a great number that will shoot at the eye if it gets close enough. I believe this is a required quest to get beyond the starting zone for Death Knights. The quest adds two additional buttons, an "Eye of Archerus" and "Cancel". Clicking Cancel does return you to the starting point of the quest. Clicking the other has no affect.

I hopped on retail and noticed that the entire bottom bar changes out with eyeball-specific spells, whereas on TC the bar continues to contain your toon's own items.

TrinityCore rev. 4695e23 2015-12-09 01:33:37 +0100 (6.x branch) (Unix, Release)
Database: TDB 6.03
